lib/jsonapi/fetchdata/parameters/parsers/paginate.rb

```diff
@@ -8,7 +8,10 @@ class Paginate < Parser
 
           def parse params={}
             number, limit, size = params.values_at('number', 'limit', 'size')
-            { 'number': (number || 1).to_i, 'size': (size || limit || Kaminari.config.max_per_page || 40).to_i }
+            {
+              number: (number || 1).to_i,
+              size: (size || limit || Kaminari.config.max_per_page || 40).to_i
```

Comment:
Any preferences how to keep Kaminari dependency out. Config object for settings?
```ruby
JSONAPI.configure do |config|
  config.max_page_size = Kaminari.config.max_per_page
end
```